meta.yml

```diff
@@ -44,18 +44,25 @@ license:
   identifier: MIT
 
 supported_coq_versions:
-  text: 8.9 or later
-  opam: '{(>= "8.9" & < "8.14~") | (= "dev")}'
+  text: 8.10 or later
+  opam: '{(>= "8.10" & < "8.14~") | (= "dev")}'
 
 tested_coq_nix_versions:
 - version_or_url: https://github.com/coq/coq-on-cachix/tarball/master
 
 tested_coq_opam_versions:
-- version: dev
-- version: '8.12'
-- version: '8.11'
-- version: '8.10'
-- version: '8.9'
+- version: 'coq-dev'
+  repo: 'mathcomp/mathcomp-dev'
+- version: '1.12.0-coq-8.13'
+  repo: 'mathcomp/mathcomp'
+- version: '1.12.0-coq-8.12'
+  repo: 'mathcomp/mathcomp'
+- version: '1.11.0-coq-8.12'
+  repo: 'mathcomp/mathcomp'
+- version: '1.10.0-coq-8.11'
+  repo: 'mathcomp/mathcomp'
+- version: '1.9.0-coq-8.10'
+  repo: 'mathcomp/mathcomp'
```

Comment:
I don't know the semantics of this field, but it looks a bit like this says we require mathcomp, when really we don't (just for one of the tests). Is that a potential problem?

Reply:
The way the `all` target works is that it will always require `coq-mathcomp-ssreflect` to be installed. To save time in CI, I select Docker images that already have the ssreflect package installed along with Coq. I doubt people will read the CI configuration to figure out dependencies (this is what the README and opam packages are for).

Reply:
I saw the CI config changes in `.github/workflows/coq-ci.yml`; how is `meta.yml` related to that?

Reply:
I use our [configuration templates](https://github.com/coq-community/templates) to generate `.github/workflows/coq-ci.yml` based on the data in `meta.yml`. The boilerplate in the target yaml file is based on best practices for using the Coq Docker Action and maintained centrally. This is our response to the ever-changing configuration file changes on GitHub and elsewhere.